src/main/java/bio/terra/cli/context/TerraUser.java

```diff
@@ -1,6 +1,6 @@
-package bio.terra.cli.model;
```

Comment:
This is a tough one – I could see it fitting here in the `context` package, or over in `auth`. Arguably, `auth` may be ever so slightly better as it would reduce the number of cross-package dependencies (since TerraUser depends on AuthUtils, and AuthManager depends on TerraUser).

Reply:
I prefer having all the classes that define how we persist context on disk in the same place. So other packages look to one place to see the state available to them. If we end up with a lot more context-related classes, then I might go the other way -- distributing them among the other functional packages. I think all the other packages (except server-related commands) will depend on TerraUser, since most of them get the current user as part of their operations, so having auth as the one exception doesn't seem like a large benefit to me.